build/webpack/webpack.extension.web.config.js

```diff
@@ -12,8 +12,7 @@ const prodEntry = {
     extension: './src/extension.web.ts'
 };
 const devEntry = {
-    extension: './src/extension.web.ts',
-    'test/index': './src/test/web/index.ts' // source of the web extension test runner
```

Comment:
I don't think this entry point is useful anymore.

Reply:
Aren't you writing to this file below? I thought this means that index.ts is included in the bundle too.

Reply:
No its not included in the bundle, when we have multiple entry points such as `extension` and `test/index` then we end up with multiple bundles.